Box.V2/Managers/BoxFoldersManager.cs

```diff
@@ -250,7 +253,11 @@ public BoxFoldersManager(IBoxConfig config, IBoxService service, IBoxConverter c
         /// attributes can be passed in separated by commas e.g. fields=name,created_at. Paginated results can be 
         /// retrieved using the limit and offset parameters.
         /// </summary>
-        /// <returns></returns>
+        /// <param name="id">This param is not used in implementation</param>
+        /// <param name="limit">The maximum number of items to return</param>
+        /// <param name="offset">The maximum number of items to return</param>
```

Comment:
This method is Obsolete and input param "id" is not used and is not sent as param and does not matter what value includes.